leakcanary-android-core/src/main/java/leakcanary/internal/activity/db/HeapAnalysisTable.kt

```diff
@@ -92,8 +94,7 @@ internal object HeapAnalysisTable {
     db.inTransaction {
       return db.rawQuery(
           """
-              SELECT
-              object
+              SELECT object
```

Comment:
One thing to test here is that leaks that were recorded prior to this update can still be displayed after this update

Reply:
I just tested by installing the sample app on the existing main branch and then installing it with my changes.
The old leaks can still be displayed, where the only issue is that heapdurationmillis for the old leaks will be 0ms.
My hypothesis is that the object deserialization does not recognize the default argument in Kotlin, which seems to be okay unless you would prefer 0 and -1 to both be illegal values.

Reply:
Thanks for testing! Sounds good.